lib/dpdk/iceBoardStandard.hpp

```diff
@@ -152,15 +164,30 @@ inline bool iceBoardStandard::advance_frame(uint64_t new_seq, bool first_time) {
     // Advance the frame
     if (!first_time) {
         mark_frame_full(out_buf, unique_name.c_str(), out_frame_id);
-
-        // Advance frame ID
         out_frame_id = (out_frame_id + 1) % out_buf->num_frames;
+
+        // Advance the lost samples frame
+        mark_frame_full(lost_samples_buf, unique_name.c_str(), lost_samples_frame_id);
+        lost_samples_frame_id = (lost_samples_frame_id + 1) % lost_samples_buf->num_frames;
```

Comment:
Can we change the type on the frame IDs to `frameID`? To simplify these lines to:
```
  mark_frame_full(lost_samples_buf, unique_name.c_str(), lost_samples_frame_id++);
```

Reply:
We likely should, but all of the DPDK related code uses this style, so I think it should be changed all at once, and is thus out of scope for this PR.